docs/example/basic/custom-ipv4-subnet.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: spiderpool.spidernet.io/v2beta1
+kind: SpiderSubnet
+metadata:
+  name: custom-ipv4-subnet
+spec:
+  subnet: 172.18.41.0/24
+  ips:
+    - 172.18.41.40-172.18.41.50
```

Comment:
This code patch appears to be defining a Kubernetes resource of kind `SpiderSubnet` with the name `custom-ipv4-subnet`. The subnet specified in the IPv4 address range `172.18.41.0/24`, and has been configured with IP addresses `172.18.41.40` through `172.18.41.50`.

Without more context, it's difficult to comment on whether this code will work as intended or not. However, a few general suggestions:

1. Make sure that the `apiVersion` is correct and up-to-date for the Kubernetes version being used.
2. When specifying IP addresses, ensure that they do not conflict with any existing IP addresses in use.
3. Consider adding more metadata to the resource (e.g., labels) to make it more identifiable and manageable.

Overall, the code looks concise and straightforward, but as always, there's room for improvement with more context about the specific use case.